### PR TITLE
MOSIP-14289 RegClient: invalid input Parameter-individual type displayed during onboarding and update operator biometric

### PR DESCRIPTION
**Pull request by neosoftSatish**

### CODE REVIEW

File: registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/InternalAuthDelegateServiceImpl.java [hunk 1]
```diff
@@ -82,6 +84,7 @@ public AuthResponseDTO authenticate(AuthRequestDTO authRequestDTO, HttpHeaders h
 		String individualId = getIndividualIdByUserId(authRequestDTO.getIndividualId());
 
 		authRequestDTO.setIndividualId(individualId);
+		authRequestDTO.setIndividualIdType(UIN);
```
Comment (vishwa-vyom) [inline registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/InternalAuthDelegateServiceImpl.java]: As per the operator validator code, setIndividualIdType was sent as UIN ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/InternalAuthDelegateServiceImpl.java (hunk 1 above)
Comment (neosoftSatish) [reply registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/InternalAuthDelegateServiceImpl.java]: in operator validator, when we are fetching individualId... it is setting individualType to null...

Comment (neosoftSatish) [reply registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/InternalAuthDelegateServiceImpl.java]: if (INDIVIDUAL_TYPE_USERID.equalsIgnoreCase(individualType)) {
			userId = getIndividualIdByUserId(userId);
			individualType = null;
		}

Comment (vishwa-vyom) [reply registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/InternalAuthDelegateServiceImpl.java]: Then the same should work here, the internal API has the intellegence to find the type of Id if null is sent.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**